tasks.py

```diff
@@ -444,3 +444,16 @@ def _replace(pattern):
     run("git commit -a -m 'Automated update for release v{}'".format(version), echo=True)
     run("git tag v{} -m 'See the release notes for details:\n\nhttps://metallb.universe.tf/release-notes/#version-{}-{}-{}'".format(version, version.major, version.minor, version.patch), echo=True)
     run("git checkout main", echo=True)
+
+@task
+def unit_tests(ctx):
+    """Run unit tests."""
+    run("go test ./...")
+    run("go test -race ./...")
+
+@task
+def linter(ctx):
+    """Run linter."""
+    # Install "gometalinter" and run linter
+    run("curl -L https://git.io/vp6lP | sh")
```

Comment:
Is there a container we could run that includes this?  It's one thing to do this automatically in the CI job, but some people might be surprised that we install stuff on their dev machine automatically like this.

Reply:
Possibly. I looked at your PR, and like you I was just copying the CI build instructions. I am happy to investigate some improvements like you suggest above but it looks like your PR is near to merge. Perhaps I should wait for merge and then build on top of it.